apps/api/src/app/inbox/usecases/mark-many-notifications-as/mark-many-notifications-as.command.ts

```diff
@@ -15,4 +16,9 @@ export class MarkManyNotificationsAsCommand extends EnvironmentWithSubscriber {
   @IsOptional()
   @IsBoolean()
   readonly archived?: boolean;
+
+  @IsOptional()
+  @Type(() => Date)
+  @IsDate()
+  readonly snoozedUntilDate?: Date | null;
```

Comment:
I was thinking about using also `snoozed: boolean` to be consistent with the rest of the states, but that would mean we have 1 more unnecessary field on document that we need to maintain. This one field handles it well.

Reply:
☝️